Automattic/wp-calypso
# Privacy Settings: Hide for Unlaunched Sites

#### Changes proposed in this Pull Request

* Hides the Privacy Settings for unlaunched sites instead of just disabling it

#### Testing instructions

Visit `/settings/general/` and scroll down to the Privacy Settings. There's a couple of cases to verify. Firstly, nothing should change for launched sites. 

For unlaunched sites who haven't verified their email, that prompt should still appear with the "Launch" button.

**Before:**
<img width="865" alt="Screenshot 2020-10-25 at 14 26 22" src="https://user-images.githubusercontent.com/43215253/97110197-c25f5100-16cf-11eb-9b2a-607b1255d4b0.png">

**After:**
<img width="777" alt="Screenshot 2020-10-25 at 14 26 33" src="https://user-images.githubusercontent.com/43215253/97110239-eae74b00-16cf-11eb-8c85-d812cfdfd6a2.png">

For unlaunched sites who have verified the email, the "Launch" button should still be available. 

**Before:**
<img width="767" alt="Screenshot 2020-10-25 at 14 26 08" src="https://user-images.githubusercontent.com/43215253/97110210-d86d1180-16cf-11eb-9816-b5d1c3407fb5.png">

 **After:**
<img width="771" alt="Screenshot 2020-10-25 at 14 26 13" src="https://user-images.githubusercontent.com/43215253/97110219-df941f80-16cf-11eb-803a-86056a10939b.png">

cc @roo2, @scruffian

Fixes #46629
